src/peer_manager.rs

```diff
@@ -1217,11 +1219,14 @@ impl PeerManager {
 
         let _ = self.insert_peer(pub_id, Some(peer_id), PeerState::SearchingForTunnel);
 
-        let close_section = self.routing_table.other_close_names(pub_id.name()).unwrap_or_default();
+        let close_nodes = self.routing_table
```

Comment:
I was effectively meaning, of all the peers in our section which are direct-connected, take the 10 closest to us.  If we're concerned about always using the same peers, we could shuffle the peers in our section before filtering, but not sure if that's worthwhile or not.